.github/workflows/drivers_ci.yml

```diff
@@ -57,6 +57,15 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Install kernel headers (actuated)
+        uses: self-actuated/get-kernel-sources@master
+        if: matrix.arch == 'arm64'
+
+      - name: Install kernel headers (amd64)
+        if: matrix.arch == 'amd64'
+        run: |
+          sudo apt install -y --no-install-recommends linux-headers-$(uname -r)
+
```

Comment:
I'm not super happy with this solution: to correctly retrieve the args of the syscall and set the snaplen we need to know the syscall number. Given that we need the kernel headers to avoid defining the syscall number for each arch.